.changeset/quiet-rivers-wave.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-auth-backend': patch
+---
+
+Supports callbackUrls when setting cookie configuration.
+Cookie paths will no longer be scoped to include the handler of the provider:
+
+```diff
+cookie = {
+-  path=`${pathname}/${provider}/handler`
++  path=`${pathname}/${provider}`
```

Comment:
Bit worried this diff causes confusion tbh, we typically use diffs to show changes that integrators are supposed to make to stay up to date. Probably enough to just mention the change as you already do above.